resources/test/fixtures/isort/magic_trailing_comma.py

```diff
@@ -0,0 +1,13 @@
+from sys import (
+    stderr,
+    argv,
+    stdout,
+    exit,
+)
+
+from os import (
+    path,
+    environ,
+    execl,
+    execv
+)
```

Comment:
Can we add a few more examples, like:

```
from os import (
    path,
    environ,
    execl,
    execv,  # Ends with a comment
)
```

```
# These will be combined, but without a trailing comma.
from foo import bar
from foo import baz
```

```
# These will be combined, _with_ a trailing comma, I think? Follow `isort` here.
from foo import bar
from foo import (
  baz,
  bop,
)
```

Reply:
<img width="772" alt="Screenshot 2022-12-24 at 3 21 01 PM" src="https://user-images.githubusercontent.com/72827203/209450095-f4726f8e-be9e-48f4-aa44-360b09c65d81.png">
It looks like the way you suggested is how isort does it. This is the one test my PR is currently failing on, will work on fixing it.

Reply:
I read the isort PR at some point, and I think the logic is that if either import block has a trailing comma, they preserve that when merging the two blocks.

Reply:
Perfect! Got this implemented.